Core/Core/Todos/Todo.swift

```diff
@@ -73,9 +80,14 @@ public final class Todo: NSManagedObject, WriteableModel {
     public var subtitleText: String {
         switch type {
         case .submitting:
-            guard assignment.dueAt != nil else { return assignment.dueText }
-            let format = NSLocalizedString("Due: %@", bundle: .core, comment: "")
-            return String.localizedStringWithFormat(format, assignment.dueText)
+            guard let dueAt = assignment.dueAt else {
+                return NSLocalizedString("No Due Date", bundle: .core, comment: "")
+            }
+            let calendar = Calendar.current
+            let weekday = calendar.shortWeekdaySymbols[calendar.component(.weekday, from: dueAt)]
+            let dueText = Todo.dateFormatter.string(from: dueAt)
```

Comment:
Do you need that formatter, it doesn't look like you're doing anything custom.  Have you considered using:
```
DateFormatter.localizedString(from date: Date, 
                 dateStyle dstyle: DateFormatter.Style, 
                 timeStyle tstyle: DateFormatter.Style) -> String
```

Reply:
Oh yeah that would be better. I'll do that.

Reply:
Now I need a formatter in order to use the template.